### PR TITLE
put back the freqs_to_cBpack cutoff; prepare for 1.0

### PR DESCRIPTION
**Pull request by rspeer**

### CODE REVIEW

File: wordfreq_builder/wordfreq_builder/word_counts.py [hunk 1]
```diff
@@ -67,12 +67,16 @@ def freqs_to_cBpack(in_filename, out_filename, cutoff=-600, lang=None):
 
     Only words with a frequency greater than `cutoff` centibels will be
     written to the new file.
+
+    This cutoff should not be stacked with a cutoff in `read_freqs`; doing
+    so would skew the resulting frequencies.
     """
-    freq_cutoff = 10 ** (cutoff / 100.)
-    freqs = read_freqs(in_filename, freq_cutoff, lang=lang)
+    freqs = read_freqs(in_filename, cutoff=0, lang=lang)
```
Comment (alin-luminoso) [inline wordfreq_builder/wordfreq_builder/word_counts.py]: Nitpick: `0` is the default value anyway, isn't it?

Comment (rspeer) [inline wordfreq_builder/wordfreq_builder/word_counts.py]: Yes, just being clear about it.

File: wordfreq_builder/wordfreq_builder/word_counts.py [hunk 2]
```diff
@@ -67,12 +67,16 @@ def freqs_to_cBpack(in_filename, out_filename, cutoff=-600, lang=None):
...
     cBpack = []
     for token, freq in freqs.items():
         cB = round(math.log10(freq) * 100)
+        if cB <= cutoff:
+            continue
```
Comment (alin-luminoso) [inline wordfreq_builder/wordfreq_builder/word_counts.py]: This is `continue` here and `break` above.  Is this one expecting unordered input?

Comment (rspeer) [inline wordfreq_builder/wordfreq_builder/word_counts.py]: It's iterating over a dictionary.

Comment (alin-luminoso) [inline wordfreq_builder/wordfreq_builder/word_counts.py]: Oh.  Right.

**Pull request merged**